mailbox/api/src/main/java/org/apache/james/mailbox/acl/PositiveUserACLDiff.java

```diff
@@ -48,6 +48,10 @@ public static PositiveUserACLDiff computeDiff(MailboxACL oldACL, MailboxACL newA
             .filter(this::hasPositiveUserKey);
     }
 
+    public ACLDiff getAclDiff() {
```

Comment:
I don't agree about this changeset : we never return diff anywhere else in the code, we sometimes return the new state.
Furthermore, a Command and a Diff is likely the same thing so we can have a diff from a command easily.

Reply:
I don't understand this comment, sorry. Could you try the fix?

Reply:
I don't like it but let's say it doesn't hurt.